docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.9"
+services:
+  bash:
```

Comment:
i believe this service will still drop you into python, you'll need to add a `command` to make it give you a bash terminal instead